manifests/cf-manifest/operations.d/329-capi-release.yml

```diff
@@ -2,6 +2,6 @@
   path: /releases/name=capi
   value:
     name: "capi"
-    version: "1.81.0"
-    url: "https://bosh.io/d/github.com/cloudfoundry/capi-release?v=1.81.0"
-    sha1: "8116229c5789dc47cc6ff35d59c8a5c7e363c174"
+    version: "1.82.0"
+    url: "https://bosh.io/d/github.com/cloudfoundry/capi-release?v=1.82.0"
```

Comment:
Is this file required as it appears that [9.3 upstream is using CAPI 1.82](https://github.com/cloudfoundry/cf-deployment/blob/v9.3.0/cf-deployment.yml#L2341)?

Reply:
Good shout - I've removed this opsfile